src/main/kotlin/com/exactpro/th2/conn/dirty/fix/FixProtocolMangler.kt

```diff
@@ -115,7 +115,7 @@ class FixProtocolMangler(context: IManglerContext) : IMangler {
 
         if (rules.isEmpty()) return null
 
-        val rule = rules.filter { rule ->
+        val rule = rules.filter { it.isActiveRule }.filter { rule ->
```

Comment:
Combine two filter into one lambda, also, convert to Sequence when execute several cascade operations to avoid create intermediate collections

Reply:
removed